tools/sign_release.sh

```diff
@@ -20,6 +20,9 @@
 # If you're not creating a release but want to create a release build for local testing, you
 # can append the `--test` flag to ignore some release checks.
 
+# Crash script if any command exits with non-zero status
```

Comment:
nit: we can remove all the `|| exit 1` lines. No need to do here but we can fix on master.